Console/MyString.h

```diff
@@ -0,0 +1,24 @@
+#ifndef String_h
+#define String_h
+
+#include"BaseForType.h"
+#include<string>
+
+class MYLIBRARY_EXPORT String : public BaseForType
+{
+private:
+	std::string value;
+public:
+	String();
+	String(const std::string &val);
+	String(const String &val);
+	void Set(const std::string val);
+	void Set(std::istream &in) override;
```

Comment:
please, follow one codestyle. If you add space before 'override', do this everywhere. If class method's name begins with capital letter, make it everywhere, etc.